packages/owi/owi.0.2/opam

```diff
@@ -0,0 +1,82 @@
+opam-version: "2.0"
+synopsis:
+  "OCaml toolchain to work with WebAssembly, including an interpreter"
+description:
+  "owi is an OCaml toolchain to work with WebAssembly. It provides an interpreter as an executable and a library."
+maintainer: ["Léo Andrès <contact@ndrs.fr>"]
+authors: [
+  "Léo Andrès <contact@ndrs.fr>"
+  "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+  "Filipe Marques <filipe.s.marques@tecnico.ulisboa.pt>"
+  "Eric Patrizio <epatrizio@mpns.fr>"
+  "Arthur Carcano <arthur.carcano@ocamlpro.com"
+]
+license: "AGPL-3.0-or-later"
+tags: ["owi" "ocaml" "webassembly" "wasm" "interpreter" "compiler"]
+homepage: "https://github.com/ocamlpro/owi"
+bug-reports: "https://github.com/ocamlpro/owi/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>= "5.1"}
+  "integers" {>= "0.5.1"}
+  "cmdliner"
+  "conf-clang"
+  "conf-python-3-dev"
+  "sedlex"
+  "menhir" {build & >= "20220210"}
+  "ocaml_intrinsics"
+  "uutf"
+  "bisect_ppx" {with-test & >= "2.5" & dev}
+  "ocb" {with-test & >= "0.1" & dev}
+  "odoc" {with-doc}
+  "crunch" {dev}
+  "bos"
+  "smtml"
+  "mdx" {with-test & >= "2.1"}
+  "crowbar" {with-test}
+  "graphics" {dev}
+  "tiny_httpd" {dev}
+  "ocamlformat" {dev}
+  "digestif"
+  "xmlm"
+  "pyml"
+  "re2"
+  "hc" {>= "0.3"}
+  "dune-site"
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["dune" "install" "-p" name "--create-install-files" name]
+]
+dev-repo: "git+https://github.com/ocamlpro/owi.git"
+available: (arch = "x86_32" | arch = "x86_64" | arch = "arm64") & os != "win32"
+depexts: [
+  [ "llvm" "lld" "wabt"]  {os-family = "debian"}
+  [ "llvm" "wabt"]  {os-family = "homebrew"}
+  [ "llvm" "lld" "wabt" ] {os-family = "arch"}
+  [ "llvm" "lld" "wabt" ] {os-family = "fedora"}
+  [ "llvm" "lld" ] {os-family = "opensuse"}
+  [ "llvm" "lld" "wabt" ] {os-family = "bsd"}
+]
+pin-depends: [
```

Comment:
This still has a pin-depends ... that needs to be removed before it can go into mainline opam-repo.

Reply:
Actually it is only a dev dependency and is not part of the automatic testsuite, the fuzzer has to be run manually so I'll remove it. :)